civiform/civiform
# [Program Card Images] On the admin UI, disable the submit buttons when there haven't been any changes

### Description

The program image upload UI has two submit buttons: One to save the description and one to save the image. But, those buttons were always enabled even if the user didn't change the description or didn't upload an image. This PR disables the buttons until the admin changes the description or adds an image.

## Release notes

### Checklist

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)


#### User visible changes


### Instructions for manual testing

1. Enable the PROGRAM_CARD_IMAGES flag.
2. As an admin, edit a program then click "Edit program image".
3. Verify the "Save image description" and "Save image" buttons are initially disabled.
4. Type a description and verify the "Save image description" button becomes enabled.
5. Save the description.
6. Upload an image and verify the "Save image" button becomes enabled.

### Issue(s) this completes

Fixes #6307 

Part of epic #5676
